### ISSUE TITLE
How to print the class number in the detection?

### ISSUE THREAD

**Issue by fdkssdks** [pierluigiferrari/ssd_keras#176]
* Operating System Ubuntu 16.04
    * Which commit of this repository you're on: Latest
    * Keras version: 2.2.0
    * TensorFlow version: 1.9.0
I've this function where I can print all the data in the list using function print(y_pred_thresh[0]).
```
Predicted boxes:

   class   conf    xmin   ymin   xmax   ymax  

[[  21.      0.73   81.57  156.7   102.19  181.04]]
```
I can't figure out how to print just the class number for ex 21.

```
graph = tf.get_default_graph()
		y_pred = self.model._make_predict_function()
		y_pred = self.model.predict(input_images)
		#change below to see more or less predictions
		confidence_threshold = 0.66

		y_pred_thresh = [y_pred[k][y_pred[k,:,1] > confidence_threshold] for k in range(y_pred.shape[0])]

		np.set_printoptions(precision=2, suppress=True, linewidth=90)
		print("Predicted boxes:\n")
		print('   class   conf    xmin   ymin   xmax   ymax ', "\n")
		print(y_pred_thresh[0])
#To draw bounding box		
for box in y_pred_thresh[0]:
		objects+=1
		# Transform the predicted bounding boxes for the 300x300 image to the original image dimensions.
		xmin = box[2] * 300 / img_width
		ymin = box[3] * 300 / img_height
		xmax = box[4] * 300 / img_width
		ymax = box[5] * 300 / img_height
		#cv2.rectangle(img, (x1, y1), (x2, y2), (255,0,0), 2)
		cv2.rectangle(smallFrame, (int(xmin),int(ymin)),(int(xmax),int(ymax)),(0,250,0),thickness=1,lineType=8)
		#wait to finish writing
		cv2.waitKey(0)
```
Thanks

**Comment by lgov** [comment]
Hi, 

the `box` variable is a list of 6 values:

`(label, conf, xmin, ymin, xmax, ymax) = box`

So the original label value that you are looking for is stored in box[0].


If you look at the inference tutorial:
https://github.com/pierluigiferrari/ssd_keras/blob/master/ssd300_inference.ipynb

in chapter "4. Visualize the predictions", you can find these relevant lines:
```python
label = '{}: {:.2f}'.format(classes[int(box[0])], box[1])
current_axis.add_patch(plt.Rectangle((xmin, ymin), xmax-xmin, ymax-ymin, color=color, fill=False, linewidth=2))
current_axis.text(xmin, ymin, label, size='x-large', color='white', bbox={'facecolor':color, 'alpha':1.0})
```
... which looks up the label value `box[0]` in the `classes` list to created a formatted string with the class (e.g. car) and confidence, and then draw that on the image.

hth,
Lieven

**Comment by fdkssdks** [comment]
Thanks I got this working by printing box[0].
Im closing this now
